scripts/xcode_build.rb

```diff
@@ -51,7 +51,7 @@
 
 # link static libraries
 target.frameworks_build_phases.clear
-libs = ['libc10.a', 'libclog.a', 'libnnpack.a', 'libeigen_blas.a', 'libcpuinfo.a', 'libpytorch_qnnpack.a', 'libtorch.a']
+libs = ['libc10.a', 'libclog.a', 'libnnpack.a', 'libeigen_blas.a', 'libcpuinfo.a', 'libpytorch_qnnpack.a', 'libtorch_cpu.a', 'libtorch.a']
```

Comment:
Do we still want to keep `libtorch.a`? If I understand correctly, we only need one which is `libtorch_cpu.a`

Reply:
libtorch.a is a stub library that doesn't contain any actual code. So it's harmless if we include it or not.

Reply:
don't know what `libtorch.a` exactly contains. But if it doesn't contain any actual code, can we just skip it? Would there be any error without linking it?

Reply:
I think we can skip it. I'm just kind of worried about what might happen if we do add code to the library at some point in the future: then we better not skip it!

Reply:
lol.  I guess you're right